lib/core/test/bench/db/Main.hs

```diff
@@ -629,12 +632,16 @@ testPk :: PrimaryKey WalletId
 testPk = PrimaryKey testWid
 
 ourAccount :: ShelleyKey 'AccountK XPub
-ourAccount = publicKey $ unsafeGenerateKeyFromSeed (seed, mempty) mempty
-  where seed = Passphrase $ BA.convert $ BS.replicate 32 0
+ourAccount = publicKey $ unsafeGenerateKeyFromSeed (seed, Nothing) mempty
+  where
+    seed = SomeMnemonic $ entropyToMnemonic @15 ent
+    ent = either (error . show) id $ mkEntropy $ BS.replicate 32 0
 
 rewardAccount :: ShelleyKey 'AddressK XPub
-rewardAccount = publicKey $ unsafeGenerateKeyFromSeed (seed, mempty) mempty
-  where seed = Passphrase $ BA.convert $ BS.replicate 32 0
+rewardAccount = publicKey $ unsafeGenerateKeyFromSeed (seed, Nothing) mempty
+  where
+    seed = SomeMnemonic $ entropyToMnemonic @15 ent
+    ent = either (error . show) id $ mkEntropy $ BS.replicate 32 0
```

Comment:
I'd suggest to move this to `Cardano.Wallet.Unsafe` under `unsafeSomeMnemonic`  with a type parameter to apply.

Reply:
I added `unsafeSomeMnemonicFromEntropy`.

Reply:
The export list now looks like:
-    ( unsafeFromHex
-    , unsafeFromHexFile
-    , unsafeDecodeAddress
-    , unsafeDecodeHex
-    , unsafeFromText
-    , unsafeRunExceptT
-    , unsafeXPrv
-    , unsafe**Mk**Mnemonic
-    , unsafe**Mk**Entropy
-    , unsafeSomeMnemonicFromEntropy
-    , unsafeDeserialiseCbor
-    , unsafeBech32DecodeFile
-    , unsafeBech32Decode

I guess `unsafeSomeMnemonicFromEntropy` should be `unsafeMkSomeMnemonicFromEntropy`.

Reply:
Done.